core/src/main/java/org/eclipse/hono/connection/ConnectionFactoryImpl.java

```diff
@@ -127,7 +127,8 @@ private void handleConnectionAttemptResult(
 
         if (conAttempt.failed()) {
 
-            logger.debug("can't connect to AMQP 1.0 container [{}://{}:{}]: {}", clientOptions.isSsl() ? "amqps" : "amqp",
+            logger.debug("can't connect to AMQP 1.0 container [{}://{}:{}]: {}",
```

Comment:
can you simply remove this change from this PR?

Reply:
yes I will